public/plugins/grafana-testdata-datasource/src/plugin.json

```diff
@@ -1,13 +1,15 @@
 {
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
   "type": "datasource",
   "name": "TestData",
-  "id": "testdata",
+  "id": "grafana-testdata-datasource",
```

Comment:
Are we sure we want to change the name of plugins? I know that `grafana-XXX-datasource` is used by all expernal plugins, but this change is going to break dashboards/stats?

Reply:
yes, it's necessary for publishing plugins in grafana.com. It shouldn't break dashboards because it's setting the old ID as an alias: https://github.com/grafana/grafana/pull/74104/files#diff-3e54b9e2506d1e2760013e8b618d86da4f2b1332858f1e50b18fce533b07a2a9R85

If we discover this breaks something, we should fix the alias behavior for that case.

Reply:
Thank you for explanation @andresmgot! 🧡